browser/main/Detail/FolderSelect.styl

```diff
@@ -134,54 +134,37 @@ body[data-theme="dark"]
   .search-optionList-item-name-surfix
     color $ui-dark-inactive-text-color
 
-body[data-theme="monokai"]
- .root
-    color $ui-dark-text-color
-    &:hover
-      color white
-      background-color $ui-monokai-button--hover-backgroundColor
-      border-color $ui-monokai-borderColor
-
-  .search-optionList
-    color white
-    border-color $ui-monokai-borderColor
-    background-color $ui-monokai-button-backgroundColor
-
-  .search-optionList-item
-    &:hover
-      background-color lighten($ui-monokai-button--hover-backgroundColor, 15%)
-
-  .search-optionList-item--active
-    background-color $ui-monokai-button--active-backgroundColor
-    color $ui-monokai-button--active-color
-    &:hover
-      background-color $ui-monokai-button--active-backgroundColor
-      color $ui-monokai-button--active-color
-  .search-optionList-item-name-surfix
-    color $ui-monokai-inactive-text-color
-
-body[data-theme="dracula"]
- .root
-    color $ui-dracula-text-color
```

Comment:
I think you miss this line in the rewrite? Or we don't need it? :smile:

Reply:
Ya, there is an issue with the template.
But there is another issue, the theme `solarized-dark` is different from the master.
The master use the `white` theme whereas the current one use `solarized-dark` colors.